trikot-viewmodels/sample/common/src/commonMain/kotlin/com/trikot/viewmodels/sample/viewmodels/home/ViewsViewModel.kt

```diff
@@ -18,20 +18,42 @@ class ViewsViewModel : MutableListViewModel<ListItemViewModel>() {
 
     override var elements: Publisher<List<ListItemViewModel>> = listOf<ListItemViewModel>(
         MutableHeaderListItemViewModel(".backgroundColor"),
-        MutableViewListItemViewModel().also {
-            it.view.backgroundColor = StateSelector(Color(255, 0, 0)).just()
+        MutableViewListItemViewModel().apply {
+            view.backgroundColor = StateSelector(Color(255, 0, 0)).just()
         },
         MutableHeaderListItemViewModel(".alpha"),
-        MutableViewListItemViewModel().also {
-            it.view.alpha = 0.5f.just()
+        MutableViewListItemViewModel().apply {
+            view.alpha = 0.5f.just()
         },
         MutableHeaderListItemViewModel(".hidden"),
-        MutableViewListItemViewModel().also {
-            it.view.hidden = true.just()
+        MutableViewListItemViewModel().apply {
+            view.hidden = true.just()
         },
         MutableHeaderListItemViewModel(".onTap"),
-        MutableViewListItemViewModel().also {
-            it.view.action = ViewModelAction { navigationDelegate?.showAlert("Tapped $it") }.just()
+        MutableViewListItemViewModel().apply {
+            view.action = ViewModelAction { navigationDelegate?.showAlert("Tapped $it") }.just()
+        },
+        MutableHeaderListItemViewModel(".isAccessibilityElement (true)"),
+        MutableViewListItemViewModel().apply {
+            view.accessibilityLabel = "This should be accessible".just()
+            view.isAccessibilityElement = true.just()
+        },
+        MutableHeaderListItemViewModel(".isAccessibilityElement (false)"),
+        MutableViewListItemViewModel().apply {
+            view.accessibilityLabel = "This should not be accessible".just()
+            view.isAccessibilityElement = false.just()
+        },
+        MutableHeaderListItemViewModel(".accessibilityLabel (\"Sample view\")"),
+        MutableViewListItemViewModel().apply {
+            view.accessibilityLabel = "This is a sample view".just()
+            view.isAccessibilityElement = true.just()
+        },
+        MutableHeaderListItemViewModel(".accessibilityHint (\"Purchase the item.\")"),
+        MutableViewListItemViewModel().apply {
+            view.accessibilityLabel = "Purchase".just()
+            view.accessibilityHint = "Purchase the item".just()
```

Comment:
I would add a test using a behaviourSubject that actually changes the accessibility components after purchasing, just to showcase that these can be updated dynamically.

Reply:
Good thing you mentioned this because I realized that `accessibilityHint` lacked flexibility. 
Updating the `accessibilityLabel` worked great but the new `accessibilityHint` was not announced right away when it was modified. (We had to unfocus and focus again to know the new value)

I've made some changes that allow us to specify the new hint, also state if we want the new value to be announced immediately and if we want a `customHintsChangeAnnouncement` just for this moment.

So for example the normal `hint` can be "Purchase the item" / "Cancel your purchase" and the `customHintsChangeAnnouncement` can be "Interact again to cancel your purchase".

https://user-images.githubusercontent.com/6673075/152411967-633d5a36-20b9-43c0-ac8b-2c4f9cd2127e.mp4

https://user-images.githubusercontent.com/6673075/152411996-4d3ecbe5-0a5b-4781-b89a-5638dc3eabe5.mp4